crates/ruff_linter/resources/test/fixtures/airflow/AIR303.py

```diff
@@ -0,0 +1,23 @@
+from airflow.operators.python import PythonOperator
+from airflow import DAG
+
+# Using deprecated task_concurrency parameter
+task1 = PythonOperator(
+    task_id="task1",
+    task_concurrency=2,  # This should trigger AIR303
```

Comment:
@abhishekbhakat We should raise a deprecation error in Airflow 2.2+ too: https://github.com/apache/airflow/pull/17708 (that was when we had original deprecation)

In Airflow 3, it is "removed" entirely.